src/snowflake/snowpark/mock/functions.py

```diff
@@ -231,7 +236,7 @@ def mock_listagg(column: ColumnEmulator, delimiter, is_distinct):
     columns_data = ColumnEmulator(column.unique()) if is_distinct else column
     return ColumnEmulator(
         data=delimiter.join([str(v) for v in columns_data.dropna()]),
```

Comment:
Returns a string that includes all of the non-NULL input values, separated by the delimiter.

Reply:
this is for commenting purpose?

Reply:
Yes and I also think the implementation is not filtering the NULL values out 🤔